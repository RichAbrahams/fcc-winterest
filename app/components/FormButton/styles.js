import styled from 'styled-components';

export const Button = styled.button`
  margin: 1em 0em 0em 0em;
  padding: 0.5em;
  background: #1b81ca;
  color: white;
  border-radius: 3px;
  width: ${(props) => props.width};
  &:hover {
    background: rgb(88, 185, 6);
    cursor: pointer;
  }
  @media screen and (max-width: 500px) {
    width: 100%;
  }
`;
