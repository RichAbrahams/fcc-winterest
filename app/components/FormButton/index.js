/**
*
* FormButton
*
*/

import React from 'react';
import { Button } from './styles';


function FormButton(props) {
  return (
    <Button>
      {props.children}
    </Button>
  );
}

FormButton.propTypes = {
  children: React.PropTypes.string,
};

export default FormButton;
