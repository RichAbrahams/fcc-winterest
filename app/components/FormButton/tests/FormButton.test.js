import React from 'react';
import renderer from 'react-test-renderer';
import FormButton from '../index';

it('renders correctly', () => {
  const tree = renderer.create(
    <FormButton>test</FormButton>).toJSON();
  expect(tree).toMatchSnapshot();
});
