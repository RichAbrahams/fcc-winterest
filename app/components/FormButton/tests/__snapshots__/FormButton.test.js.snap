exports[`test renders correctly 1`] = `
<button
  className="kdIDvS">
  test
</button>
`;
